config/environments/development.rb

```diff
@@ -15,4 +15,10 @@
   config.assets.raise_runtime_errors = true
 
   config.mx_checker = MxChecker::Dummy.new
+  config.smoke_test_email_local_part = ENV.fetch(
+    'SMOKE_TEST_EMAIL_LOCAL_PART',
+    'smoke-test')
+  config.smoke_test_email_domain = ENV.fetch(
+    'SMOKE_TEST_EMAIL_DOMAIN',
+    'example.com')
```

Comment:
The wrapping here feels a bit odd. Perhaps this would be better:

```ruby
config.smoke_test_email_domain =
  ENV.fetch('SMOKE_TEST_EMAIL_DOMAIN', 'example.com')
```